tests/test_butler.py

```diff
@@ -649,6 +654,9 @@ def tearDown(self):
         bucket = s3.Bucket(self.bucketName)
         bucket.delete()
 
+        # unset any potentially set dummy credentials
+        unsetAwsEnvCredentials()
```

Comment:
I think you want an `if self.usingDummyCredentials` statement here (like you have done in the other tests).

Reply:
You are right, my bad. Thanks for the quick review.